src/odemis/util/dataio.py

```diff
@@ -189,3 +201,20 @@ def _split_planes(data):
         das.append(plane)
 
     return das
+
+def open_acquisition(filename, fmt=None):
+    if fmt:
+        converter = dataio.get_converter(fmt)
+    else:
+        converter = dataio.find_fittest_converter(filename, mode=os.O_RDONLY)
+    data = []
+    try:
+        if hasattr(converter, 'open_data'):
+            acd = converter.open_data(filename)
+            data = [(c, acd, i) for i, c in enumerate(acd.content)]
```

Comment:
No, do no use AcquisitionData+index, but just use DataArrayShadows (now that they have eveything on them).